### PR TITLE
Issue #2502: Add line break after The Jupyter Notebook is running at:

### PR DESCRIPTION
**Pull request by GadgetSteve**
Updated both notebook/notebookapp.py and docs/source/security.rst to add a newline before the URL - testing showed that the URL still did not fit in 80 columns because of the `[I 11:59:16.597 NotebookApp]` leader - changed to a `\n\r` to get the correct results. - Dropped the \ at the end idea as the logger output makes this pointless.

Tested that the clipboard gives the correct results. 

Minor change to currently untested feature so no test cases added. Existing tests still pass with one exception - `ERROR: Failure: ImportError (No module named fcntl)` which is as before for me.

Note: A number of lines in notebook/notebookapp.py were changed due to trailing spaces the only actual changes are on lines 1356-1357.

### CODE REVIEW

File: notebook/notebookapp.py [hunk 1]
```diff
@@ -226,7 +226,7 @@ def init_settings(self, jupyter_app, kernel_manager, contents_manager,
             },
             version_hash=version_hash,
             ignore_minified_js=jupyter_app.ignore_minified_js,
-
```
Comment (takluyver) [inline notebook/notebookapp.py]: Could we ask you to redo this with your editor set not to strip whitespace? Sorry for the hassle, but the changes can cause unnecessary merge conflicts, and make tools like `git blame` less useful.

Other than this, it looks great, thanks :-)

**Comment by GadgetSteve** [comment]
Will do and I might raise a separate ticket for stripping whitespace.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by takluyver** [comment]
Thanks! Be aware that we don't generally go for 'cleanup' PRs.

**Comment by takluyver** [comment]
Thanks :-)

**Pull request merged**